test/Sentry.Testing/SentryOptionsExtensions.cs

```diff
@@ -0,0 +1,70 @@
+namespace Sentry.Testing;
+
+/// <summary>
+/// We most frequently call IDiagnosticLogger via extension methods on the SentryOptions class, which obscures what
+/// we're expecting to receive via mocks when writing tests. This class provides some test extensions that mirror the
+/// extensions provided for SentryOptions to make it easier to write tests that match the code they're testing.
+/// </summary>
+public static class SentryOptionsExtensions
+{
+    private static SentryOptions DidNotReceiveReceiveLog(this SentryOptions substitute, SentryLevel level)
```

Comment:
Is it named ReceivedReceived on purpose?
```suggestion
    private static SentryOptions DidNotReceiveLog(this SentryOptions substitute, SentryLevel level)
```